src/basecases.py

```diff
@@ -156,3 +93,105 @@ def bootstrap_dependencies_seven_nodes():
     demo_nodes[6].dependency = ItemDependency([screws], awesomeness)
 
     return demo_nodes
```

Comment:
this whole section needs to go right?

Reply:
I added this into main. I commented it out, so you can easily change main.py and add this.